pkg/netlink/netlink.go

```diff
@@ -47,8 +47,7 @@ type Basic interface {
 	RouteGet(destination net.IP) ([]netlink.Route, error)
 	RouteList(link netlink.Link, family int) ([]netlink.Route, error)
 	FlushRouteTable(tableID int) error
-	RuleAdd(rule *netlink.Rule) error
-	RuleDel(rule *netlink.Rule) error
+	ConfigureIPRule(operation Operation, tableID int) error
```

Comment:
In think we should keep the `RuleAdd` and `RuleDel` primitives as is rather than overloading a single function to do both. Also we can add a shortcut function to create a table rule:

```
func NewTableRule(tableID int) *netlink.Rule {
    rule := netlink.NewRule()
    rule.Table = tableID
    rule.Priority = tableID

    return rule
}
```

Reply:
OK, I think we can keep to RuleAdd and RuleDel primitives

I'll update the PR